tests/upgrades/test_capsule.py

```diff
@@ -112,57 +111,51 @@ def test_post_user_scenario_capsule_sync(self):
             2. The repos/rpms from satellite should be synced to satellite
 
         """
-        env_name = get_entity_data(self.__class__.__name__)['env_name']
-        org_name = entities.Organization().search(query={'search': f'id={self.org_id}'})[0].label
-        capsule = entities.SmartProxy().search(query={'search': f'name={self.cap_host}'})[0]
+        request.addfinalizer(lambda: __cleanup(content_view, repo, product))
+        org = entities.Organization().search(query={'search': f'name="{DEFAULT_ORG}"'})[0]
+        pre_test_name = dependent_scenario_name
+        rpm_name = rpm1.split('/')[-1]
+        cap_host = settings.upgrade.rhev_cap_host or settings.upgrade.capsule_hostname
+        activation_key = settings.upgrade.rhev_capsule_ak or settings.upgrade.capsule_ak
+        ak = entities.ActivationKey(organization=org.id).search(
+            query={'search': f'name={activation_key}'}
+        )[0]
+        repo = entities.Repository(organization=org.id).search(
+            query={'search': f'name={pre_test_name}_repo'}
+        )[0]
+        product = entities.Product(organization=org.id).search(
+            query={'search': f'name={pre_test_name}_prod'}
+        )[0]
+        env_name = ak.environment.read()
+        org_name = env_name.organization.read_json()['label']
+
+        content_view = entities.ContentView(organization=f'{org.id}').search(
+            query={'search': f'name={pre_test_name}_cv'}
+        )[0]
+        capsule = entities.SmartProxy().search(query={'search': f'name={cap_host}'})[0]
         call_entity_method_with_timeout(entities.Capsule(id=capsule.id).content_sync, timeout=3600)
         result = execute(
             lambda: run(
-                '[ -f /var/lib/pulp/published/yum/http/repos/'
-                '{}/{}/{}/custom/{}/{}/Packages/b/{} ]; echo $?'.format(
-                    org_name, env_name, self.cv_name, self.prod_name, self.repo_name, self.rpm_name
-                )
+                f'[ -f /var/lib/pulp/published/yum/http/repos/'
+                f'{org_name}/{env_name.name}/{content_view.name}/custom/{pre_test_name}_prod/'
+                f'{pre_test_name}_repo/Packages/b/{rpm_name} ]; echo $?'
             ),
-            host=self.cap_host,
-        )[self.cap_host]
-        self.assertEqual('0', result)
+            host=cap_host,
+        )[cap_host]
+        assert result == '0'
 
 
-class Scenario_capsule_sync_2(APITestCase):
+class TestCapsuleSync2:
```

Comment:
If you're going to keep classes in the test module, their names should be more meaningful. Maybe `TestCapsuleSyncNewRepo` - or just drop the classes altogether.

Reply:
I'm not convinced that this file needs classes, or at least not this many.

Reply:
@mshriver @JacobCallahan   Agree, I will change the name and bind the two post-upgrade scenarios in a single class as there are no pre-upgrade test cases for that.
 
Class is required here, It binds the pre and post-upgrade test cases in a single scenario and that will help others to understand what this scenario does.